scripts/setup_al2.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set +u
+
+if [ -z "$1" ]; then
+    echo "When running this script, please supply the name of the user account for which to set up unstructured dependencies."
```

Comment:
I thought about whether if someone ran this with no argument it should default to installing just for themselves, but decided not to because I constantly run the script with no argument when that's not what I mean to do. 

My reason for setting it up this way in the first place is that the script began life as an EC2 setup script, which runs as root but needs to set things up for the non-root user. We could drop that paradigm here?

Reply:
Yeah I think that's a good safeguard against this. And there's a clear error message if you run it without a username. Good to go from my end!

> with no argument when that's not what I mean to do.